docs/network_firewall.rst

```diff
@@ -465,20 +469,37 @@ to add a rule.
 Once you've set up the firewall, exit the Unsafe Browser, and continue
 with the "Keeping pfSense up to date" section below.
 
-Configuration Reference Templates
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+SecureDrop Configuration (Templates)
+------------------------------------
+
+As an alternative to the manually provided screenshots, you can use our ``.xml``
+templates. However, because of the Tails chroot, you can not download the
+``.xml`` files directly. Using the Unsecure Browser, download the files from
+`GitHub <https://github.com/freedomofpress/securedrop/tree/develop/install_files/network_firewall/>`__
+as "Raw" and save them as plaintext documents.
```

Comment:
Wait, why are we asking folks to download files from the develop branch? These files should be local on disk already—and have their integrity validated with the signed git tag from the latest release.